pkg/webui/console/containers/shortcut-panel/index.js

```diff
@@ -32,7 +32,7 @@ const m = defineMessages({
   shortcuts: 'Quick actions',
   addApplication: 'New Application',
   addGateway: 'New Gateway',
-  addOrganization: 'New Organization',
+  addNewOrganization: 'New Organization',
```

Comment:
```suggestion
  addNewOrganization: 'New organization',
```
There was a mistake in the designs, the labels should use normal sentence case.